pkg/failsaferules/failsaferules.go

```diff
@@ -26,6 +24,18 @@ var tcp = []TransportProtoFailSafeRule{
 		"SSH",
 		22,
 	},
+	{
```

Comment:
are we still using this with failsafe rules being gone?

Reply:
its gone as rules but not gone from webhook validation ?